[2022-12-25T19:38:36.107+0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_pivot.first_task scheduled__2022-06-23T15:00:00+00:00 [queued]>
[2022-12-25T19:38:36.119+0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_pivot.first_task scheduled__2022-06-23T15:00:00+00:00 [queued]>
[2022-12-25T19:38:36.120+0300] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-25T19:38:36.120+0300] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2022-12-25T19:38:36.121+0300] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-25T19:38:36.800+0300] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): first_task> on 2022-06-23 15:00:00+00:00
[2022-12-25T19:38:36.806+0300] {standard_task_runner.py:55} INFO - Started process 27017 to run task
[2022-12-25T19:38:36.815+0300] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'titanic_pivot', 'first_task', 'scheduled__2022-06-23T15:00:00+00:00', '--job-id', '366', '--raw', '--subdir', 'DAGS_FOLDER/first_dag_total.py', '--cfg-path', '/var/folders/g7/wpcjf399379f4cptqxrgszxr0000gn/T/tmphd_io_av']
[2022-12-25T19:38:36.820+0300] {standard_task_runner.py:83} INFO - Job 366: Subtask first_task
[2022-12-25T19:38:37.309+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:38:37.312+0300] {logging_mixin.py:137} WARNING - /Users/vadim/opt/anaconda3/lib/python3.9/configparser.py:408 FutureWarning: section/key [core/sql_alchemy_conn] has been deprecated, you should use[database/sql_alchemy_conn] instead. Please update your `conf.get*` call to use the new name
[2022-12-25T19:38:37.317+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/settings.py:400 DeprecationWarning: The sql_engine_encoding option in [core] has been moved to the sql_engine_encoding option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:38:37.319+0300] {logging_mixin.py:137} WARNING - /Users/vadim/opt/anaconda3/lib/python3.9/configparser.py:408 FutureWarning: section/key [core/sql_engine_encoding] has been deprecated, you should use[database/sql_engine_encoding] instead. Please update your `conf.get*` call to use the new name
[2022-12-25T19:38:37.603+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/utils/sqlalchemy.py:121 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:38:37.680+0300] {task_command.py:389} INFO - Running <TaskInstance: titanic_pivot.first_task scheduled__2022-06-23T15:00:00+00:00 [running]> on host mbp-vadim.bbrouter
[2022-12-25T19:38:38.157+0300] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=titanic_pivot
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-23T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-23T15:00:00+00:00
[2022-12-25T19:38:38.159+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /var/folders/g7/wpcjf399379f4cptqxrgszxr0000gn/T
[2022-12-25T19:38:38.161+0300] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "Here we start!"']
[2022-12-25T19:38:38.174+0300] {subprocess.py:86} INFO - Output:
[2022-12-25T19:38:38.366+0300] {subprocess.py:93} INFO - Here we start!
[2022-12-25T19:38:38.371+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-25T19:38:38.696+0300] {taskinstance.py:1322} INFO - Marking task as SUCCESS. dag_id=titanic_pivot, task_id=first_task, execution_date=20220623T150000, start_date=20221225T163836, end_date=20221225T163838
[2022-12-25T19:38:38.779+0300] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-25T19:38:39.063+0300] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
